.travis.yml

```diff
@@ -0,0 +1,11 @@
+dist: xenial   # required for Python >= 3.7
+language: python
+python:
+  - "3.7"
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
```

Comment:
You actually don't need to do this, as `requirements-tests.txt` already has `-r requirements.txt` in it.